fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -36,7 +36,14 @@ class SKSCTestImpl : public SpecialKeyRangeBaseImpl {
 		ASSERT(resultFuture.isReady());
 		auto result = resultFuture.getValue();
 		ASSERT(!result.more);
-		return resultFuture.getValue();
+		// To make the test more complext, instead of simply returning the k-v pairs, we reverse all the value strings
```

Comment:
I'm not sure what value this adds

Reply:
I think the special key space contract is that: 
The implementation of a special key can return f(key), where f(key) is a function that can do more operations than simply return the key. 

The operation here is to simulate that contract I think.